libs/ledger-live-common/src/families/bitcoin/js-synchronisation.ts

```diff
@@ -149,11 +149,12 @@ const getAccountShape: GetAccountShape = async (info) => {
   span.finish();
 
   const oldOperations = initialAccount?.operations || [];
-  await wallet.syncAccount(walletAccount);
-  const balance = await wallet.getAccountBalance(walletAccount);
-  const currentBlock = await walletAccount.xpub.explorer.getCurrentBlock();
+  const [, currentBlock] = await Promise.all([
+    wallet.syncAccount(walletAccount),
```

Comment:
Could you remove the await in the loop in the sync method so we don't have to disable an eslint rule ?

// eslint-disable-next-line no-await-in-loop

Thanks !

Reply:
fixed